cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,67 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
+  const userName = 'Bohdan';
+  const userSurname = 'Svystun';
+  const email = 'randomemail@gmail.com';
+  const subject = 'No subjects';
```

Comment:
The 'No subjects' may not be valid input. Please check if it's a valid subject or use a valid subject instead.